Header header
int64 request_number
string request_type
string active_state
